packages/discovery-styles/scss/components/search-input/_search-input.scss

```diff
@@ -1,7 +1,3 @@
-// Component styles
-@import '~carbon-components/scss/components/search/search';
```

Comment:
does `@use` work here? https://css-tricks.com/introducing-sass-modules/#:~:text=The%20new%20%40use%20is%20similar,considered%20private%2C%20and%20not%20imported.

or can we at least leave a comment indicating which carbon styles this file depends on?

Reply:
It seems like it would work, but we would have to use `dart-sass` instead of `node-sass` which then requires anyone consuming this package to also use `dart-sass`. 

At this point in time, it doesn't appear that `@use` is on the `node-sass` roadmap: https://github.com/sass/node-sass/issues/2886#issuecomment-608280859

Reply:
@maniax89 would you prefer comments per file or references within this file: https://github.com/watson-developer-cloud/discovery-components/blob/62561bc372d6d19c4d1a91ce1e12414cac1a0137/packages/discovery-styles/scss/global-carbon-styles.scss#L12

Reply:
either or - i don't have a preference. just as long as it is consistent

Reply:
Seems like many components import the same base styles. For example

https://github.com/watson-developer-cloud/discovery-components/blob/master/packages/discovery-styles/scss/components/ci-document/_ci-document-filter.scss#L1-L3

and 

https://github.com/watson-developer-cloud/discovery-components/blob/master/packages/discovery-styles/scss/components/search-facets/_search-facets.scss#L1-L3

Thinking that I'll go the route of putting all of the comments in the global file and include a section of shared base carbon styles across components.

Reply:
Did that here: https://github.com/watson-developer-cloud/discovery-components/pull/146/files#diff-bb1e94d2444f11bfc036c8c60f806d24